packages/fxa-settings/src/components/Settings/BentoMenu/en.ftl

```diff
@@ -1,7 +1,9 @@
 # BentoMenu component
 
 bento-menu-title = { -brand-firefox } Bento Menu
+bento-menu-title-2 = { -brand-mozilla } Bento Menu
 bento-menu-firefox-title = { -brand-firefox } is tech that fights for your online privacy.
+bento-menu-mozilla-title = { -brand-mozilla } is tech that fights for your online privacy.
```

Comment:
@LZoog - One of our localizers raised an issue with this text. I can see what this person means, would we want to consider changing the string content here, or possibly replace `{ -brand-mozilla }` with something else? (Mozilla accounts?)

>This confuses me then- is it correct to say "Mozilla is tech"? Previously "Firefox is tech" made sense, but now I'd expect this to say "Mozilla is a tech organization" or such

Reply:
@bcolsson I missed this comment and will raise that now.

Reply:
Confirmed that for now we're going to _leave_ this string as `Firefox is tech...` and revisit this soon.

Reply:
Thakns, understood.